portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,18 @@
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  font-family: sans-serif
 }
 
-#greeting-container {
+#fun-fact-container {
   margin-top: 20px;
 }
+
+.centered {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 70%;  
```

Comment:
design nit: You probably want to put a max-width on this in PX just so it doesn't get ridiculously wide on large screens.

Reply:
Nevermind, found it

Reply:
Thanks! Added.